src/components/People.js

```diff
@@ -0,0 +1,71 @@
+import "../App.css";
+
+import React, { Component } from 'react'
+
+class People extends Component {
+    constructor(){
+        super()
+        this.state={
+            characterData:[],
+            userInput: ""
+        }
+
+    }
+
+    fetchCharacters=()=>{
+        fetch("https://ghibliapi.herokuapp.com/people")
+        .then(res=> res.json())
+        .then((data)=>{
+            this.setState({
+                characterData: data
+            })
+        })
+    }
+
+    componentDidMount(){
+        this.fetchCharacters()
+    }
+
+    handleUserInput=(event)=>{
+        this.setState({
+            userInput: event.target.value
+        })
+    }
+    handleSubmit=(event)=>{
+        event.preventDefault()
+     
+    }
+
+
+    render() {
+            let selectedCharactersObj = this.state.characterData.find((character)=>{
+                return character.name.toLowerCase() === this.state.userInput.toLowerCase() 
+            })
+
```

Comment:
```suggestion
let characterInfo;
    if (selectedCharactersObj) {
      characterInfo = (
        <div className="character-container">
          <div>{selectedCharactersObj.name}</div>
          <div>{selectedCharactersObj.age}</div>
          <div>{selectedCharactersObj.gender}</div>
        </div>
      );
    } else {
      characterInfo = <div>Not Found</div>;
    }
```
In order to **not** display "Not Found" before the user searches for anything, you have to do a couple things. Like I mentioned before, your conditional logic needs another layer to it to account for the three options of what may be rendered to the page:
1) **null**, if **NO** search has been made yet
2) **character info JSX**, if a search **has** been made and it was _successful_
3) **JSX that says "Not Found"** if a search **has** been made and it was _unsuccessful_

This is one way to add that missing piece of conditional logic you're missing. This variable called `characterInfo` will be displayed in place of lines 57-62, and there will be a ternary operator as well to check if any search has been made at all yet.